third_party/nvfuser/csrc/scheduler/normalization.cpp

```diff
@@ -21,15 +21,178 @@ namespace nvfuser {
 
 namespace {
 
-// round up to multiple of 8 or pow2 whichever smaller
-int64_t roundUpPow2Or8(const int64_t x) {
-  auto round_up_pow2 = scheduler_utils::lastPow2(x);
-  if (round_up_pow2 < x) {
-    round_up_pow2 *= 2;
+std::shared_ptr<ReductionParams> innerOuterPersistentHeuristic(
+    const int64_t total_reduction_numel,
```

Comment:
Please comment what `total_reduction_numel` means in the combined case. Doesn't it mean the iteration number of elements for the outer reduction, do it?

Reply:
deleted as it is not used.

Reply:
renamed `inner_most_dimension_numel` to `inner_dim_numel` and
`total_iteration_numel` to `outer_dim_numel`